fast_matched_filter/fast_matched_filter.py

```diff
@@ -174,6 +174,11 @@ def test_matched_filter(n_templates=1, n_stations=1, n_components=1,
     """
     output: templates, moveouts, data, step, cc_sum
     """
+    if float(int(sampling_rate)) != float(sampling_rate):
+        print('The sampling rate should be an integer !\nCorrect that before running the test function.')
+        return
+    else:
+        sampling_rate = int(sampling_rate)
 
```

Comment:
I'm thinking we should check that the durations are a multiple of the sampling interval instead of checking for an integer sampling rate.